<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrayWidthForm</class>
 <widget class="QDialog" name="ArrayWidthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creating array data</string>
  </property>
  <widget class="QPushButton" name="AddBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Please enter the number of column of your array</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfColumnSBx">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
